docs/DISTROS.md

```diff
@@ -0,0 +1,150 @@
+- curl patches: [list of patches in package git](https://src.fedoraproject.org/rpms/curl/tree/rawhide)
+
+## Gentoo Linux
+
+- curl: https://packages.gentoo.org/packages/net-misc/curl
+- curl issues: https://bugs.gentoo.org/buglist.cgi?quicksearch=net-misc/curl
+- curl package sources and patches: https://gitweb.gentoo.org/repo/gentoo.git/tree/net-misc/curl/
+
+## Homebrew
+
+- curl: https://formulae.brew.sh/formula/curl
+
+Homebrew's policy is that all patches should be submitted upstream unless it
+is very specific to Homebrew's way of packaging software. The issues could
+also just point to the generic Homebrew issue tracker. There is no specific
+curl issue tracker in Homebrew.
+
+## Mageia
+
+curl: https://svnweb.mageia.org/packages/cauldron/curl/current/SPECS/curl.spec?view=markup
+curl issues: https://bugs.mageia.org/buglist.cgi?bug_status=NEW&bug_status=UNCONFIRMED&bug_status=NEEDINFO&bug_status=UPSTREAM&bug_status=ASSIGNED&component=RPM%20Packages&f1=cf_rpmpkg&list_id=176576&o1=casesubstring&product=Mageia&query_format=advanced&v1=curl
+curl patches: https://svnweb.mageia.org/packages/cauldron/curl/current/SOURCES/
+
+## MSYS2
+
+- curl: https://github.com/msys2/MINGW-packages/tree/master/mingw-w64-curl
+- curl issues: https://github.com/msys2/MINGW-packages/issues
+- curl patches: https://github.com/msys2/MINGW-packages/tree/master/mingw-w64-curl (`*.patch`)
+
+# Muldersoft
+
+- curl: https://github.com/lordmulder/cURL-build-win32
+- curl issues: https://github.com/lordmulder/cURL-build-win32/issues
+- curl patches: https://github.com/lordmulder/cURL-build-win32/tree/master/patch
+
+## NixOS
+
+- curl: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/networking/curl/default.nix
+- curl issues: https://github.com/NixOS/nixpkgs
+
+nixpkgs is the package repository used by the NixOS Linux distribution, but
+can also be used on other distributions
+
+## OmniOS
+
+- curl: https://github.com/omniosorg/omnios-build/tree/master/build/curl
+- curl issues: https://github.com/omniosorg/omnios-build/issues
+- curl patches: https://github.com/omniosorg/omnios-build/tree/master/build/curl/patches
+
+## OpenIndiana
+
+- curl: https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/components/web/curl
+- curl issues: https://www.illumos.org/projects/openindiana/issues
+- curl patches: https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/components/web/curl/patches
+
+## Oracle Solaris
+
+- curl: https://github.com/oracle/solaris-userland/tree/master/components/curl
+- curl issues: https://support.oracle.com/ (requires support contract)
+- curl patches: https://github.com/oracle/solaris-userland/tree/master/components/curl/patches
+
+## pkgsrc
+
+- curl: https://github.com/TritonDataCenter/pkgsrc/tree/trunk/www/curl
+- curl issues: https://github.com/TritonDataCenter/pkgsrc/issues
+- curl patches: https://github.com/TritonDataCenter/pkgsrc/tree/trunk/www/curl/patches
```

Comment:
Better to use the main NetBSD repository at https://github.com/NetBSD/pkgsrc, but otherwise correct.